source/release-notes/4-1-x-breaking-changes.rst

```diff
@@ -0,0 +1,67 @@
+:is-up-to-date: True
+:last-updated: 4.1.0
+:orphan:
+
+.. _breaking-changes-4-1-x:
+
+====================================
+Breaking Changes in CrafterCMS 4.1.x
+====================================
+This section covers changes that might affect your CrafterCMS projects, as well as other considerations
+before upgrading. Please review the following and apply changes as required:
+
+- CrafterCMS 4.1.x requires Java 17. |br| See :ref:`Requirements <requirements_supported_platforms>` for more
+  information on CrafterCMS requirements.
+
+- CrafterCMS 4.1.x now uses OpenSearch. |br| See :ref:`here <upgrading-search>` for more information on upgrading search to OpenSearch
+
+.. _compatibility-with-3.1.x:
+
+- **Upgrading 3.1.x sites** |br|
+
+  - CrafterCMS 4.1.x requires Git. |br| See :ref:`Requirements <requirements_supported_platforms>` for more
+    information on CrafterCMS requirements.
+
+  - CrafterCMS 4.1.0 has a new Studio UI. To get the same Sidebar you're used to, update
+    the :ref:`user-interface-configuration`
+
+  - Update ICE to move to :ref:`XB <experience-builder>`
+
+  - Studio and Engine SAML configuration now uses a key and certificate for configuration instead of using keystore. |br|
+    See :ref:`engine-saml2-configuration` and :ref:`crafter-studio-configure-studio-saml` for more information.
+
+|hr|
+
+.. _breaking-changes-4-1-3:
+
+------------------------------------
+Breaking Changes in CrafterCMS 4.1.3
+------------------------------------
+The following breaking changes applies to CrafterCMS version 4.1.3
+
+^^^^^^^^^
+Studio UI
+^^^^^^^^^
```

Comment:
Add note:

This is primarily an internal change. It would only affect those developing authoring applications using the functions described below either via `@craftercms/studio-ui` npm package or using the `craftercms` global on an authoring extension.